### PR TITLE
libudev.h shouldn't be required with ozw 1.6

### PR DESCRIPTION
**Pull request by darkbasic**
It's not needed to install libudev-dev in order to compile open-zwave

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,12 +118,9 @@ kubectl apply -k https://raw.githubusercontent.com/OpenZWave/Zwave2Mqtt/master/k
 
 ### NodeJS or PKG version
 
-1. Firstly you need to install [Open-Zwave](https://github.com/OpenZWave/open-zwave) library on your system.
-
-   If you are using Ubuntu:
```
Comment (robertsLando) [inline README.md:123]: Why did you delete this?

Comment (darkbasic) [reply README.md:123]: I didn't delete it, I just added the world "the" before "Open-ZWave library" so the line has been replaced with a new one.

**Review by robertsLando — APPROVED** [review]
LGTM
